src/Services/cdl/VarInfo.js

```
@@ -0,0 +1,28 @@
+ * ADLI tool (lineNo, colNum, varid etc...).It also saves the filename and
+ * file path. 
+ */
+class VarInfo {
+    /**
+     * @param {Object} varInfo Variable info.
+     * @param {Number} logType Logtype this variable belogns to.
+     */
+    constructor (varInfo, logType) {
+        for (var key in varInfo) {
+            this[key] = varInfo[key];
+        }
```

ESLint

[error] The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.

(guard-for-in)